django_emarsys/apps.py

```diff
@@ -0,0 +1,14 @@
+from django.apps import AppConfig
+from django.core.checks import register
+
+from django_emarsys.config import validate_settings
+
+
+class DjangoEmarsysConfig(AppConfig):
+    name = 'django_emarsys'
+    verbose_name = "Django Emarsys"
+
+    def ready(self):
+        @register()
+        def check_settings(app_configs):
+            return validate_settings()
```

Comment:
Registering checks in `AppConfig.ready()` is indeed the recommended way.

Unfortunately, `ready()` also is called for tests, so let's just accept that even the test run will validate the given settings.